src/components/CameraControls/index.tsx

```diff
@@ -228,10 +229,12 @@ const CameraControls = ({
         </div>
     );
 
-    const renderControls = minimalControls
-        ? minimalEmbeddedControls
-        : fullControls;
-
-    return renderControls;
+    if (displayType === CameraControlsDisplay.None) {
+        return null;
+    } else if (displayType === CameraControlsDisplay.Min) {
+        return minimalEmbeddedControls;
+    } else {
+        return fullControls;
+    }
```

Comment:
since you already had conditional rendering it made sense to do it all in once place. alternatively you could make a different component and push the conditional rendering up higher.